recipes/pharokka/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.6" %}
+{% set name = "pharokka" %}
+{% set sha256 = "def2acc81b0610a0b25d196c3a48069c35daefd334dd3a24243b551b9eb99dc7" %}
+{% set user = "gbouras13" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+
+source:
+  url: https://github.com/{{ user }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+ run:
```

Comment:
these dependencies are very specific and could prevent users from installing this tool in the same environment as other tools. Are these exact versions required, or can you relax them? E.g., `biopython>=1.78` rather than `==1.78`, and so on.

Reply:
Requirements should be able to be relaxed as you say, especially for biopython.

Reply:
Latest commit should have resolved this

Reply:
yep, thanks